src/fluent/utils.ts

```diff
@@ -115,7 +115,7 @@ export async function k8sCfg(method: FetchMethods) {
   });
 
   // Enable compression
-  opts.compress = true;
+  opts.compress = false;
```

Comment:
this seems like a really good thing to try too--but we might consider testing these changes independently

Reply:
Agreed, this is left over from the test where this was the only change. [GH Issue](https://github.com/defenseunicorns/pepr/issues/1203) [Soak Result](https://github.com/defenseunicorns/pepr/actions/runs/11125582719/job/30913654890#step:16:757) did not show a difference. 

However, in this specific case, we are not using compress at all. We are only using opts to get the cluster [ca-cert](https://github.com/defenseunicorns/kubernetes-fluent-client/blob/9dfb5686d43ee480e4d687532e0cd904d5d57f3e/src/fluent/watch.ts#L385) (cert and key too if in dev) and manufacturing our own agent.